gitlab/resource_gitlab_group_test.go

```diff
@@ -62,6 +62,88 @@ func TestAccGitlabGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccGitlabGroup_subgroup(t *testing.T) {
+	var group gitlab.Group
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabGroupDestroy,
+		Steps: []resource.TestStep{
+			// Create a group with subgroup_creation_level set to owner
+			{
+				Config: testAccGitlabGroupConfig(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabGroupExists("gitlab_group.foo", &group),
+					testAccCheckGitlabGroupAttributes(&group, &testAccGitlabGroupExpectedAttributes{
+						Name:                  fmt.Sprintf("foo-name-%d", rInt),
+						Path:                  fmt.Sprintf("foo-path-%d", rInt),
+						Description:           "Terraform acceptance tests",
+						SubGroupCreationLevel: "owner",
+					}),
+				),
+			},
+			// Update the group to change the level to maintainer
+			{
+				Config: testAccGitlabGroupUpdateConfig(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGitlabGroupExists("gitlab_group.foo", &group),
+					testAccCheckGitlabGroupAttributes(&group, &testAccGitlabGroupExpectedAttributes{
+						SubGroupCreationLevel: "maintainer",
+					}),
+				),
+			},
+		},
+	})
+}
+
+func TestAccGitlabGroup_projects(t *testing.T) {
+	var group gitlab.Group
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabGroupDestroy,
+		Steps: []resource.TestStep{
+			// Create a group with project_creation_level set to noone
+			{
+				Config: testAccGitlabGroupConfig(rInt),
```

Comment:
You're supposed to also update the terraform configuration in that function to add the tested fields https://github.com/claranet/terraform-provider-gitlab/pull/20/files#diff-8b16c976fb8080e1dfe284518ed9bd22L295

Reply:
I knew I missed something, fixed  in latest commit